code/modules/projectiles/guns/energy/laser.dm

```diff
@@ -8,6 +8,32 @@
 	ammo_type = list(/obj/item/ammo_casing/energy/lasergun)
 	ammo_x_offset = 1
 	shaded_charge = 1
+	var/obj/item/external_lens/stored = null
+
+/obj/item/gun/energy/laser/attackby(obj/item/I, mob/user, params)

```

Comment:
Given this is on base path with no checks, this allows lasertag/practice guns and other weird subtypes to be turned into fully functional guns.